MANIFEST.in

```diff
@@ -1,3 +1,4 @@
+exclude docker-compose.yml
```

Comment:
Why exclude? It's part of the source repo…

Reply:
@wsanchez I think we're unclear on what should be included. Is this a production distribution package? If so the docker-compose isn't really set up for production just yet, but could be.